.changeset/social-pots-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@omnigraph/openapi': patch
+---
+
+Fix the HATEOAS href anonymizationn
```

Comment:
When there are two operations in the OAS;
`/products/` and `/products/{id}`
Due to the bug in the logic, they conflict and the linking can choose the first one without any argument.